### PR TITLE
Dev-repo_cleanup models and losses docstrings

### PR DESCRIPTION
**Pull request by taraspiotr**
- Minor reformatting in docstrings
- Refactored loss functions in models.py
- Docstrings for ResNet and VGG16 U-Net models
- Docstrings for loss functions in models.py

### CODE REVIEW

**Comment by taraspiotr** [comment]
@jakubczakon @apyskir  I was not sure, if losses docstrings should contain exact definitions of loss functions, since they are only wrappers for lower level loss functions, where they are (or should be) documented in detail, for example in nn.CrossEntropy(). What do you think?

File: src/models.py [hunk 1]
```diff
@@ -136,13 +140,42 @@ def callbacks_unet(callbacks_config):
...
+    If weights_function is not None weights are calculated by applying this function on target[:, 1:, :, :].
+    If weights_function is None weights are taken from target[:, 1, :, :].
+
+    Args:
+        output (torch.Tensor): Model output of shape (N x C x H x W).
+        target (torch.Tensor): Target of shape (N x (1 + K) x H x W). Where K is number of different weights.
+        weights_function (function, optional): Function applied on target for weights.
+
+    Returns:
+        float: Loss value.
```
Comment (jakubczakon) [inline src/models.py]: @taraspiotr are you sure pytorch 0.3X version has this as a float and not tensor?

File: src/models.py [hunk 2]
```diff
@@ -136,13 +140,42 @@ def callbacks_unet(callbacks_config):
...
+
+def get_weights(target, w0, sigma, imsize):
     '''
-    w1 is temporarily torch.ones - it should handle class imbalance for thw hole dataset
+    w1 is temporarily torch.ones - it should handle class imbalance for the whole dataset
     '''
-    distances = target[:, 1, :, :]
-    sizes = target[:, 2, :, :]
-    target = target[:, 0, :, :].long()
+    w0, sigma, C = _get_loss_variables(w0, sigma, imsize)
```
Comment (jakubczakon) [inline src/models.py]: Way cleaner this way I like it.

File: src/models.py [hunk 3]
```diff
@@ -206,7 +277,9 @@ def dice_loss(output, target, smooth=0, activation='softmax'):
     loss = 0
     dice = DiceLoss(smooth=smooth)
     output = activation_nn(output)
-    for class_nr in range(1, int(target.max()) + 1):
+    for class_nr in range(output.size(1)):
```
Comment (jakubczakon) [inline src/models.py]: @taraspiotr Thanks for fixing that. I think we didn't see it here because case where entire batch doesn't have 1 class is rare but it could easily happen in the next project.

File: src/models.py [hunk 4]
```diff
@@ -172,30 +203,70 @@ def _get_size_weights(sizes, C):
...
+    """Calculate Dice Loss for multiple class output.
+
+    Args:
+        output (torch.Tensor): Model output of shape (N x C x H x W).
+        target (torch.Tensor): Target of shape (N x H x W).
+        smooth (float, optional): Smoothing factor. Defaults to 0.
+        activation (string, optional): Name of the activation function, softmax or sigmoid. Defaults to 'softmax'.
+        excluded_classes (list, optional):
+            List of excluded classes numbers. Dice Loss won't be calculated
+            against these classes. Defaults to [].
```
Comment (jakubczakon) [inline src/models.py]: @taraspiotr Maybe example like (often used on background class) would show why/when to use it.

File: src/postprocessing.py [hunk 5]
```diff
@@ -81,11 +81,11 @@ def erode_image(mask, erode_selem_size):
     """Erode mask.
 
     Args:
-        mask (numpy.ndarray): mask of shape (H x W) or set of masks of shape (C x H x W)
-        erode_selem_size (int): size of rectangle structuring element used for erosion
+        mask (numpy.ndarray): Mask of shape (H x W) or set of masks of shape (C x H x W).
```
Comment (jakubczakon) [inline src/postprocessing.py]: @taraspiotr do you think that  `set` could be confusing and `list of masks` is better?

File: src/models.py (hunk 1 above)
Comment (taraspiotr) [reply src/models.py]: @jakubczakon you are right.

File: src/postprocessing.py (hunk 5 above)
Comment (taraspiotr) [reply src/postprocessing.py]: @jakubczakon `list` and `set` are both python built-ins so maybe `multiple masks`?
